editor/components/rich-text/index.js

```diff
@@ -130,6 +133,14 @@ export class RichText extends Component {
 		};
 
 		this.containerRef = createRef();
+
+		if ( ! coreTokensRegistered ) {
+			forOwn( tokens, ( { name, settings } ) => {
+				registerToken( name, settings );
```

Comment:
I think with the way `withDispatch` work, it's not certain that `registerToken` is available in the constructor. Maybe more in `componentDidMount`

Reply:
Why is that so? Why only so late in `componentDidMount`?

Reply:
I'm not certain about this assumption honestly :). I was thinking the proxy methods created by `withDispatch` will be available later but looking at the code now, it seems that I was wrong. So nvm this one

Reply:
Hm, I'm not understanding why it's not working in the tests specifically... Not sure what I'm missing here. :/